Resources/input/fsh/extensions/GEM_ERP_EX_LastMedicationDispense.fsh

```diff
@@ -0,0 +1,18 @@
+Extension: GEM_ERP_EX_LastMedicationDispense
+Id: GEM-ERP-EX-LastMedicationDispense
+Title: "Last Medication Dispense"
+Description: "This extension should be used in the Task ressource. It shows the timestamp of the last dispense."
+* ^url = "https://gematik.de/fhir/erp/StructureDefinition/GEM_ERP_EX_LastMedicationDispense"
+* ^version = "1.3.0"
+* ^status = #draft
+* . ^short = "Timestamp which is set when a medication for this task has been dispensed. Must be stated in the format YYYY-MM-DDTHH:MM:SS(Z|±HH:MM)"
+* . ^definition = "This field allows the E-Rezept-FdV to display to the user that a medication has been dispensed before it was closed by the pharmacy."
+* ^context.type = #element
+* ^context.expression = "Task"
+* value[x] only instant
+* value[x] obeys workflow-letzteAbgabeDatumsFormat
+
+Invariant:   workflow-letzteAbgabeDatumsFormat
+Description: "The 'value' field must match the datetime format up to seconds precision YYYY-MM-DDTHH:MM:SS(Z|±HH:MM)"
+Expression: "value.matches('^[0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])T([01][0-9]|2[0-3]):[0-5][0-9](:[0-5][0-9]|60)?(Z|([+-](0[0-9]|1[0-3]):[0-5][0-9]|14:00))$')"
```

Comment:
Ich habe den Regex mit verschiedenen Varianten ausprobiert und mit dem FHIR-Standard verglichen. Ist hier beabsichtigt, dass das Format analog zu dem FHIR-Standard funktioniert oder gibt es hier eine bewusste Abweichung?

Hier meine Tests der Validität im Vergleich bei beiden

| Information | FHIR | Regex |
| :--- | :----: | :----: |
| 2023 | X | |
| 2023-12 | X | |
| 2023-12-31 | X | |
| 2023-12-31T | | |
| 2023-12-31T13 | | |
| 2023-12-31T13Z | | |
| 2023-12-31T13+01:00 | | |
| 2023-12-31T13:37 | | |
| 2023-12-31T13:37Z | | X |
| 2023-12-31T13:37+01:00 | | X |
| 2023-12-31T13:37:56 | | |
| 2023-12-31T13:37:56Z | X | X |
| 2023-12-31T13:37:56+01:00 | X | X |
| 2023-12-31T13:37:56-01:00 | X | X |

Reply:
Ich habe den Regex vom FHIR instant type genommen und den Millesekunden teil gelöscht
`^([0-9]([0-9]([0-9][1-9]|[1-9]0)|[1-9]00)|[1-9]000)-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])T([01][0-9]|2[0-3]):[0-5][0-9]:([0-5][0-9]|60)(Z|(\+|-)((0[0-9]|1[0-3]):[0-5][0-9]|14:00))$`

Reply:
Habe die Anpassung in einen eigenen PR gepackt:
https://github.com/gematik/fhir-profiles-erp/pull/21